raspberry-pi/4/default.nix

```diff
@@ -16,6 +16,10 @@
 
   boot = {
     kernelPackages = lib.mkDefault pkgs.linuxPackages_rpi4;
+    kernelParams = lib.mkDefault [
```

Comment:
```suggestion
    initrd.availableKernelModules = [
```

Adding those as kernel params would do nothing.

Also note that using `mkDefault` with lists means it likely won't work as expected. Lists will generally use the unqualified priority when implemented, there is no way to compose a "`mkDefault`ed" list with one that isn't.

Reply:
Ah, the reason I'm using mkDefault is because I saw this pattern in `2/default.nix`